core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -902,6 +918,17 @@ private ExprValue exprMinute(ExprValue time) {
     return new ExprIntegerValue(time.timeValue().getMinute());
   }
 
+  /**
+   * Minute_of_day implementation for ExprValue.
+   *
+   * @param time ExprValue of Time/String type.
+   * @return ExprValue.
+   */
+  private ExprValue exprMinuteOfDay(ExprValue time) {
+    return new ExprIntegerValue(
+        time.timeValue().getHour() * MINUTES_PER_HOUR + time.timeValue().getMinute());
```

Comment:
```suggestion
        MINUTES.between(LocalTime.MIN, time.timeValue());
```

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/189/commits/956cde1cbda5d8f787fa926f1d2279359470d82a